e2e-testing/cypress/functions/SurveyConfigurationFunctions.js

```diff
@@ -52,7 +56,9 @@ class SurveyConfigurationFunctions {
   }
 
   type_post_title(title) {
-    cy.get(SurveyConfigurationLocators.postTitleField).type(title, { force: true });
+    cy.get(SurveyConfigurationLocators.postTitleField)
+      .should('be.visible')
```

Comment:
this `should('be.visible')` assertion is what we should use in place of the explicit `cy.wait` for elements to be interacted with. Here I replaced `cy.wait(3000)`
it would be a worthy effort to go through the tests and remove the explicit waits. They are not good practice in tests.

Reply:
Thanks for the correction. Will follow this from now onwards.